tox.ini

```diff
@@ -9,12 +9,16 @@ envlist = py
 
 [testenv]
 deps =
-    mock
-    jsonschema==3.1
-    git+git://github.com/DiffSK/configobj@3e2f4cc#egg=configobj
+	mock
+	flake8==3.9.2
+	# flake8-tabs version 2.3.2 gives spurious errors:
+	# "ET113 (flake8-tabs) use of alignment as indentation, but option continuation-style=hanging does not permit this"
+	flake8-tabs==2.2.2
+	-r requirements.txt
 
 commands =
-    python -m unittest discover
+	python -m unittest discover
+	flake8
```

Comment:
Just a note here to consider removing `use-pycodestyle-indent = False` to be consistent with the other discussion.